mailbox/store/src/test/java/org/apache/james/mailbox/store/mail/model/ListMailboxAssert.java

```diff
@@ -27,48 +27,28 @@
 import org.apache.james.mailbox.store.mail.model.impl.SimpleMailbox;
 
 import com.google.common.base.Function;
+import com.google.common.base.Objects;
 import com.google.common.collect.FluentIterable;
 
 public class ListMailboxAssert {
```

Comment:
Thank you for the explantions. There would be other ways, but they would be more complicated.